source/command-monitoring/command-monitoring.rst

```diff
@@ -110,7 +110,47 @@ Bulk Writes
 
 This specification defines the monitoring of inidividual commands and in that repect MUST generate
 an event for each command a bulk write executes. Each of these commands, however, must be linked
-together via the same {{operationId}}.
+together via the same ``operationId``.
+
+Implementation Notes
+--------------------
+
+The ``command`` attribute of ``CommandStartedEvent`` MUST be upconverted to the corresponding
```

Comment:
Re-reading this sentence, it's unclear what it means.  It's not the command attribute that is being up-converted, it's the legacy wire protocol messages.  This is clearer:

     Legacy wire protocol messages MUST be up-converted to the corresponding commands in order to....